system_tests/speech.py

```diff
@@ -12,23 +12,28 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 import unittest
 
 from google.cloud import exceptions
 from google.cloud import speech
 from google.cloud import storage
+from google.cloud.speech.transcript import Transcript
 
 from system_test_utils import unique_resource_id
 from retry import RetryErrors
 
 
 class Config(object):
     """Run-time configuration to be modified at set-up.
+
     This is a mutable stand-in to allow test set-up to modify
     global state.
     """
     CLIENT = None
     TEST_BUCKET = None
+    AUDIO_FILE = os.path.join(os.path.dirname(__file__), 'data', 'hello.wav')
```

Comment:
Make this a global. This isn't "Run-time configuration", i.e. it's always that value at import time, doesn't differ between the actual runs of this system test.